features/keychain/module/memoized-keychain.js

```diff
@@ -12,14 +12,14 @@ const getPublicKeyData = ({ xpub, publicKey }) => ({ xpub, publicKey })
 
 class MemoizedKeychain extends Keychain {
   #storage
-  #publicKeys = {}
+  #publicKeys = Object.create(null)
   #cloneOpts
   constructor({ storage, logger }) {
     super({ id: MODULE_ID, logger })
 
     this.#storage = storage
     this.#storage.get(CACHE_KEY).then((data) => {
-      this.#publicKeys = data ? BJSON.parse(data) : {}
+      this.#publicKeys = data ? BJSON.parse(data) : Object.create(null)
```

Comment:
@633kh4ck can we move this out into a separate `fix` PR for keychain?

Reply:
@mvayngrib, sure, will do